pkgs/top-level/unix-tools.nix

```diff
@@ -44,6 +44,13 @@ in rec {
   eject = singleBinary "eject" {
     linux = pkgs.utillinux;
   };
+  getconf = singleBinary "getconf" {
+    linux = pkgs.musl-getconf;
+    darwin = pkgs.darwin.system_cmds;
+  };
+  getent = singleBinary "getconf" {
```

Comment:
This should be `"getent"`.

Also, the musl-getconf/musl-getent (or bsd-getent/bsd-getconf)
a) Don't build w/glibc currently due to use of `strlcpy`, which means as-is this breaks all the things :(.  We may be able to pull in libbsd or something to fix.
b) May not be appropriate for use with glibc, I cannot say.  It may be worthwhile to compare `getconf -a`, and investigate `getent` differences.  I don't know if these differences matter in practice/our usage or not.

Reply:
Okay addressed in 27dd87222769bc602d34c5a5c2a13f492f432dbe.

So my hope is that any unix tools would just depend on the "kernel" - not the Libc. Definitely want to investigate whether there is a way to patch musl-* to work with glibc.